wrappingGsdk/README.md

```diff
@@ -13,106 +20,14 @@ The samples require .NET Core 3.1 SDK, you can download it [here](https://dotnet
-
-> Of course, you can still replace `fakegame` with your game server. If you choose to do so, you'll need to modify the [Dockerfile](./Dockerfile) appropriately.
-
-### Running the wrapper locally using the LocalMultiplayerAgent
-
-> Using LocalMultiplayerAgent is highly recommended if you want to test GSDK integration on your custom game servers.
-
-If you are using [LocalMultiplayerAgent](https://github.com/PlayFab/LocalMultiplayerAgent) with Windows Containers you need to properly configure `MultiplayerSettings.json` file. You can find an example below, pay special attention to the values of `LocalFilePath` and `StartGameCommand`. Don't forget to replace `fakegame.exe` with the name of your game server executable.
+- `fakegame` is a .NET Core console application. It's meant as literally a `fake game`, it just starts ASP.NET Core Web Server [Kestrel](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-3.1) that listens to TCP port. It's meant to simulate a game server that has absolutely zero knowledge of GSDK. You can use it if you don't have a game server of your own. It has two GET routes we can use, `/hello` for getting a simple response and `/hello/terminate` that can terminate the server.
 
-```json
-...
-"AssetDetails": [
-    {
-        "MountPath": "C:\\Assets",
-        "SasTokens": null,
-        "LocalFilePath": "C:\\projects\\gsdkSamples\\wrappingGsdk\\drop\\gameassets.zip"
-    }
-],
-"StartGameCommand": "C:\\Assets\\wrapper.exe -g C:\\Assets\\fakegame.exe",
-...
-// if you are using fakegameserver you should also configure port mapping for port 80
-"PortMappingsList": [
-            [
-                {
-                    "NodePort": 56100,
-                    "GamePort": {
-                        "Name": "game_port",
-                        "Number": 80,
-                        "Protocol": "TCP"
-                    }
-                }
-            ]
-        ]
-```
 
-You are now ready to test with `LocalMultiplayerAgent`! If you have configured it correctly, as soon as `LocalMultiplayerAgent` launches your game server, you can connect to it via `curl http://localhost:56100/Hello`.
+## Build Your Wrapper sample 
 
-### Next steps
+- Build and Deploy Wrapper sample as Process   
+Please check out the readme under [WrapperProcess](/ProcessSample/README.md) and learn how to build wrapper samples.
 
-You should run the [MpsAllocatorSample](../MpsAllocatorSample/README.md) to allocate/list/view Builds and game servers.
+- Create a Linux Container with Wrapper sample  
+If you want to learn how to creat a Linux build with wrapper game sample, 
```

Comment:
what about Windows containers?

Reply:
oh right.. seems we need to add more explanation to ReadMe how to run wrapper sample as Windows container.

Reply:
that's described in the instructions here https://github.com/PlayFab/MpsSamples/tree/master/wrappingGsdk#creating-the-zipped-game-assets-archive